protokt-codegen/src/main/kotlin/com/toasttab/protokt/codegen/annotators/OneofAnnotator.kt

```diff
@@ -77,13 +77,13 @@ private constructor(
                                 }
                             }
                             .addProperty(
-                                PropertySpec.builder(v.fieldName, v.type)
-                                    .initializer(v.fieldName)
+                                PropertySpec.builder(v.fieldName.removePrefix("`").removeSuffix("`"), v.type)
```

Comment:
I think this is done somewhere else too - is it worth extracting this into a utility?

Separately - I'm not sure why we have to do this. Do you know why things get escaped twice?

Reply:
I think it's a kotlinpoet bug - I'm going to look into it, and hopefully then we can get rid of this workaround.

Reply:
I think it's a kotlinpoet bug - I'm going to look into it, and hopefully then we can get rid of this workaround.

Reply:
Other instances:

https://github.com/open-toast/protokt/blob/main/protokt-codegen/src/main/kotlin/com/toasttab/protokt/codegen/annotators/MessageAnnotator.kt#L102

https://github.com/open-toast/protokt/blob/main/protokt-codegen/src/main/kotlin/com/toasttab/protokt/codegen/annotators/MessageAnnotator.kt#L128

https://github.com/open-toast/protokt/blob/main/protokt-codegen/src/main/kotlin/com/toasttab/protokt/codegen/annotators/DslAnnotator.kt#L68